syntax = "proto3";


package common;

message ErrorResponse {
  bool error = 1;
  string message = 2;
  string code = 3;
}